mobile/src/people/avatar.tsx

```diff
@@ -54,12 +53,12 @@ function validateEncodedImage(data: string) {
 }
 
 export class Avatar extends Component<AvatarProps, AvatarState> {
-    constructor(props: AvatarProps) {
-        super(props);
-        this.state = {};
-    }
+    public state: AvatarState = {
+        size: NaN,
```

Comment:
why do you use NaN? Basically it's not different from using PI. nulls should be used to indicate that no value is assigned yet.